frontend/src/tests/instructor-tests.js

```diff
@@ -18,67 +18,83 @@ import { databaseSeeder } from "./setup";
  * @param {Function} api.apiPOST A function that when passed a route and data, will return the post response
  */
 export function instructorsTests(api) {
-    const { apiGET, apiPOST } = api;
-    let instructor = null;
+    const {
+        apiGET
+        // , apiPOST
+    } = api;
+    // let instructor = null;
 
     beforeAll(async () => {
         await databaseSeeder.seed(api);
     }, 30000);
 
-    it("create instructor", async () => {
-        const newInstructorData = {
-            first_name: "Anand",
-            last_name: "Liu",
-            email: "anand.liu.sample@utoronto.ca",
-            utorid: "anandl"
-        };
+    // const newInstructorData = {
+    //     first_name: "Anand",
+    //     last_name: "Liu",
+    //     email: "anand.liu.sample@utoronto.ca",
+    //     utorid: "anandl"
+    // };
 
-        // create a new instructor
-        const resp1 = await apiPOST(`/instructors`, newInstructorData);
-        expect(resp1).toMatchObject({ status: "success" });
-        expect(resp1.payload).toMatchObject(newInstructorData);
+    it("fetch instructors", async () => {
+        const resp = await apiGET("/admin/instructors");
+        expect(resp).toHaveStatus("success");
 
-        // make sure instructor is on instructor list
-        const resp2 = await apiGET(`/instructors`);
-        expect(resp2).toMatchObject({ status: "success" });
-        expect(resp2.payload).toContainObject(newInstructorData);
+        // check the type of payload
+        checkPropTypes(PropTypes.arrayOf(instructorPropTypes), resp.payload);
 
-        // set instructor to used by later test
-        instructor = resp1.payload;
+        // compare the responsed instructor and seeded instructor they should be exactly the same
+        const responsedInstructor = resp.payload[0];
+        const seededInstructor = databaseSeeder.seededData.instructor;
+        expect(responsedInstructor).toEqual(seededInstructor);
     });
 
-    it("get instructors", async () => {
-        const resp = await apiGET("/instructors");
-        expect(resp).toMatchObject({ status: "success" });
-        checkPropTypes(PropTypes.arrayOf(instructorPropTypes), resp.payload);
-    });
+    // it("create instructor", async () => {
```

Comment:
Out of curiosity, why are these test commented out? If the purpose is to not run this test, we should probably go for `it.skip`